### PR TITLE
styling the docs a bunch

### PR DESCRIPTION
**Pull request by noahmanger**
Includes a bunch of style updates. The code is super janky and weirdly organized. I made some changes in the styles they included and some in a new "custom.less" sheet, but will clean that up later. Take a look for now and let me know how it's looking.

### CODE REVIEW

**Comment by noahmanger** [comment]
Ok, cleaned things up more or less. Style overrrides were now made in the original LESS files (either screen.less or specs.less, and any styles for new markup I added are in custom.less.

Ready for review.

File: src/main/template/main.handlebars [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
     {{/if}}]
     {{#if validatorUrl}}
-    <span style="float:right"><a href="{{validatorUrl}}/debug?url={{url}}"><img id="validator" src="{{validatorUrl}}?url={{url}}"></a>
-    </span>
+    <a href="{{validatorUrl}}/debug?url={{url}}"><img id="validator" src="{{validatorUrl}}?url={{url}}"></a>
     {{/if}}
-    </h4>
-    </div>
+    </p>
+  </\div>
```
Comment (jmcarp) [inline src/main/template/main.handlebars]: Possible typo--should this be `</div>`?

Comment (noahmanger) [inline src/main/template/main.handlebars]: Blurp. Yeah.

**Comment by LindsayYoung** [comment]
Looks good to me

**Pull request merged**